maven/bnd-resolver-maven-plugin/src/main/java/aQute/bnd/maven/resolver/plugin/ResolverMojo.java

```diff
@@ -75,6 +77,9 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 			for (File runFile : bndruns) {
 				resolve(runFile, fileSetRepository);
 			}
+		} catch (ResolutionException re) {
+			logger.error("Unresolved requirements: {}", ResolveProcess.format(re.getUnresolvedRequirements()));
+			throw new MojoExecutionException(re.getMessage(), re);
```

Comment:
This looks different than ExportMojo and TestingMojo where you nicely put it on the try/finally around the run.resolve call.